app/scripts/modules/core/src/utils/clipboard/copyToClipboard.component.ts

```diff
@@ -1,70 +0,0 @@
-import { IController, IComponentOptions, IScope, ITimeoutService, module } from 'angular';
```

Comment:
The angular component is still being used in many places.  We can't delete this file unless we make a `react2angular` adapter as well.  Because the input props should match (between the react component and the angular component), I recommend renaming the `value` prop back to `text`.

The adapter code can go in `app/scripts/modules/core/src/utils/clipboard/CopyToClipboard.component.ts` and would look like:
```
import { module } from 'angular';
import { react2angular } from 'react2angular';

import { CopyToClipboard } from './CopyToClipboard';

export const COPY_TO_CLIPBOARD_COMPONENT = 'spinnaker.core.utils.clipboard.copyToClipboard';
module(COPY_TO_CLIPBOARD_COMPONENT, []).component(
  'copyToClipboard',
  react2angular(CopyToClipboard, ['text', 'toolTip', 'analyticsLabel']),
);
```